Backend/dev/migrations/dynamic-offer-driver-app/0298-add-retarget-retries-feilds.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE atlas_driver_offer_bpp.invoice ADD COLUMN is_retried boolean DEFAULT false NOT NULL;
+ALTER TABLE atlas_driver_offer_bpp.invoice ADD COLUMN is_retargeted boolean DEFAULT false NOT NULL;
+ALTER TABLE atlas_driver_offer_bpp.invoice ADD COLUMN retarget_link Text;
+
+ALTER TABLE atlas_driver_offer_bpp.payment_order ADD COLUMN is_retried boolean DEFAULT false NOT NULL;
+ALTER TABLE atlas_driver_offer_bpp.payment_order ADD COLUMN is_retargeted boolean DEFAULT false NOT NULL;
+ALTER TABLE atlas_driver_offer_bpp.payment_order ADD COLUMN retarget_link Text;
```

Comment:
why needed in both `order` and `invoice`?

Reply:
Should it be kept only at one level ?